index.html

```diff
@@ -107,6 +107,8 @@
         <li>Complete spec text
         <li>Designated reviewers have signed off on the current spec text
         <li>The ECMAScript editor has signed off on the current spec text
+        <li><a href="https://github.com/tc39/test262">Test262</a> acceptance tests have been written for mainline usage scenarios, in a PR
```

Comment:
Just calling out that this is a specific change to the existing process

Reply:
Yep, sorry that the commit message was unclear in saying that this is a change.